packages/fdc3-agent-proxy/src/channels/DefaultChannel.ts

```diff
@@ -24,47 +23,57 @@ export class DefaultChannel implements Channel {
   }
 
   async broadcast(context: Context): Promise<void> {
-    const request: BroadcastRequest = {
-      meta: this.messaging.createMeta(),
-      payload: {
-        channelId: this.id,
-        context,
-      },
-      type: 'broadcastRequest',
-    };
-    await this.messaging.exchange<BroadcastResponse>(request, 'broadcastResponse');
+    const done = await this.messaging.exchange<BroadcastResponse>(
```

Comment:
Why convert `const request: BroadcastRequest = { ... }` to `{ ... } as BroadcastRequest`? The first one was a safer use of typescript.

Reply:
Appears to be a bad merge. Both you and I have fixed that, then it's gotten reverted when I merged the prettier PR. I'll have to comb through and look for any similar reversions (sigh)